yt/utilities/parameter_file_storage.py

```diff
@@ -114,7 +126,17 @@ def _convert_ds(self, ds_dict):
         fn = os.path.join(fp, bn)
         class_name = ds_dict["class_name"]
         if class_name not in output_type_registry:
-            raise UnknownDatasetType(class_name)
+            # first check that the relevant frontend is imported:
+            # the output_type_registry is refreshed between sessions as
+            # frontends are imported on-demand now.
+            import importlib
+
+            module = ds_dict["module_name"]
+            fe_api = ".".join(module.split(".")[0:3]) + ".api"
```

Comment:
Not 100% sure this is equivalent (I don't know how many "." are expected), but I'm willing to bet there's a clearer way to express this change in file extension.

```suggestion
            fe_api = module.rpartition(".")[0] + ".api"
```